maestro-xcuitest-driver/src/main/kotlin/util/XCRunnerSimctl.kt

```diff
@@ -81,9 +89,12 @@ object XCRunnerSimctl {
 
     fun isAppAlive(bundleId: String): Boolean {
         val process = ProcessBuilder(
-            "bash",
-            "-c",
-            "xcrun simctl spawn booted launchctl list | grep $bundleId | awk '/$bundleId/ {print \$3}'"
+            "xcrun",
```

Comment:
Is it functionally the same as before?

Reply:
The grep filters lines that contain $bundleId, awk filters so that each line only contains $bundleId.
Then two lines down, the kotlin code does:

```
val processOutput =  process.inputStream.source().buffer().readUtf8().trim()
return processOutput.contains(bundleId)
```

So yes, removing the filtering using grep and awk should functionally have no impact.